test/sql-luatest/gh_8096_seqscan_compat_test.lua

```diff
@@ -0,0 +1,70 @@
+local server = require('luatest.server')
```

Comment:
Feature tests shouldn't start with `gh_xxxx` prefix, if I remember correctly.

Reply:
Fixed, renamed the test.